src/main/java/com/fundation/webservice/model/VideoResponse.java

```diff
@@ -71,48 +58,36 @@
      * @param fps defines the rate of the output video file.
      * @param checksum defines the checksum of the output or input file.
      */
-    public VideoResponse(String fileName, String fileDownloadUri, String fileType, long size,
-                         String newFormat, String audioCodec, String audioBitRate, String audioChannel,
-                         String videoCodec, String videoBitRate, String fps, String metadata, String thumbnail,
-                         String keyframes, String checksum) {
+    public VideoResponse(String fileName, String fileDownloadUri, String fileType, long size, String newFormat,
+                         String audioCodec, String audioBitRate, String audioChannel, String videoCodec,
+                         String videoBitRate, String fps, String metadata, String thumbnail, String keyframes,
+                         String checksum) {
         this.fileName = fileName;
         this.fileDownloadUri = fileDownloadUri;
         this.fileType = fileType;
         this.size = size;
         this.newFormat = newFormat;
 
-        /**
-         * Audio constructors.
-         */
+        //Audio constructors.
         this.audioCodec = audioCodec;
         this.audioBitRate = audioBitRate;
         this.audioChannel = audioChannel;
 
-        /**
-         * Audio constructors.
-         */
+        //Audio constructors.
```

Comment:
It should say //Video constructors

I guess, :p

Reply:
Oh my god... ok my friend, correct them

Reply:
Now, I can see it is compliant.